src/apiCalls.js

```diff
@@ -2,3 +2,8 @@ export const getPokedexData = () => {
   return fetch('https://pokeapi.co/api/v2/pokemon?limit=151')
     .then(res => res.ok ? res.json() : this.displayErrorInfo(res))
 }
+
+export const getPokemonData = (suffix) => {
+  return fetch(`https://pokeapi.co/api/v2/pokemon/${suffix}`)
+    .then(res => res.ok ? res.json() : this.displayErrorInfo(res))
+}
```

Comment:
nice job starting work on incorporating a fetch to round off this logic. Eventually this should probably get called in the component did mount method for our pokemon details page.